src/test/resources/json/html.json

```diff
@@ -0,0 +1,71 @@
+[
```

Comment:
I know this is not the prefect solution as it is sometimes pain to maintain but as long as there are so many tests - tests says which steps should be updated and it's easy

I prefer to have one, complex json file with all options than having many files. Add required items to existing file (only required items